src/layout.hoc.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react'
+import { compact, getDisplayName } from './utils'
+import { SIZE } from './values'
+
+function getSize(size) {
```

Comment:
good point, is there anyway we can get some linting errors when types are missing?